src/ViewModels/test/TasExplorerViewModelTests.cs

```diff
@@ -980,32 +980,19 @@ public void SettingAuthenticationRequiredToFalse_DoesNotExpandCfInstanceViewMode
         }
 
         [TestMethod]
-        [TestCategory("DeleteConnection")]
-        public void DeleteConnection_SetsTasConnectionToNull_WhenArgIsCfInstanceViewModel()
+        [TestCategory("LogOutTas")]
+        public void LogOutTas_SetsTasConnectionToNull_WhenArgIsCfInstanceViewModel()
         {
             _sut.TasConnection = _fakeTasConnection;
 
             Assert.IsNotNull(_sut.TasConnection);
             Assert.IsTrue(_sut.TasConnection is CfInstanceViewModel);
 
-            _sut.DeleteConnection(_sut.TasConnection);
+            _sut.LogOutTas(_sut.TasConnection);
 
             Assert.IsNull(_sut.TasConnection);
```

Comment:
We should also test that `LogOutTas` sets `IsLoggedIn` to false.
```suggestion
            Assert.IsNull(_sut.TasConnection);
            Assert.IsFalse(_sut.IsLoggedIn);
```